### ISSUE TITLE
Incessant error message: "Download failed. Please check your connection." despite no connection issues.

### ISSUE THREAD

**Issue by DomD48** [mozilla/addons#1012]
When trying to install extension "I Hate Tabs - SDI", the error message "Download failed. Please check your connection." appears incessantly, despite no connection issues whatsoever: 

https://addons.mozilla.org/en-US/firefox/addon/i-hate-tabs-sdi-for-firefox/

The issue occurred at ~10:00 am ET on date 2019-05-04

**Comment by pikopikopikopiko** [comment]
seems like a bug with addons all my addons disable by default cant download any, cant install from file and so on

**Comment by LaughingLove** [comment]
There has already been multiple opened issues about this. Like #978 and #851 . You can check [this](https://discourse.mozilla.org/t/certificate-issue-causing-add-ons-to-be-disabled-or-fail-to-install/39047/13) discourse thread for updates, and check [this](https://blog.mozilla.org/addons/2019/05/04/update-regarding-add-ons-in-firefox/) blog post for some more information.

TLDR Firefox devs are aware and actively working on the issue.

**Comment by DomD48** [comment]
Okay, thanks.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by DomD48** [comment]
LaughingLove:

All 4 of the pages that you linked to feature threads that have been closed, yet this issue persists without resolution.

**Comment by caitmuenster** [comment]
Yes, as of this writing (May 4 3:00pm PST) this issue still persists, but following the blog will be the best way to keep up with updates. https://blog.mozilla.org/addons/2019/05/04/update-regarding-add-ons-in-firefox/
